### DISCUSSION TITLE
Example on how to do right and left justification on rows

### DISCUSSION THREAD

**Discussion by rnd-ash** [hecrj/iced#1207] [Q&A] [answered]
Is there any way for one to push elements to both the left and right of a Row in iced?

I have a row with 2 elements, one of which i want on the right of the window, and one on the left.

There appears to be no documentation or examples on how to do something like this.

**Comment by 90-008** [comment, marked as answer]
You can do:
```rust
let row = Row::with_children(vec![
	left_element.into(),
	Space::with_width(Length::Fill),
	right_element.into(),
]);
```
which should do the trick.

You can adjust the width etc. however you want of course. For example you may want the elements to take 1 portion and the space to take 2 portion of available width always.
